posixlib/src/main/scala/scala/scalanative/posix/unistd.scala

```diff
@@ -33,6 +33,10 @@ object unistd {
   def truncate(path: CString, length: off_t): CInt = extern
   def unlink(path: CString): CInt = extern
   def usleep(usecs: CUnsignedInt): CInt = extern
```

Comment:
I see `sleep` that take unsigned and returns unsigned - https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/unistd.h.html#tag_13_80

Reply:
Yeah, not sure how that `usleep` got there - https://newbedev.com/is-there-an-alternative-sleep-function-in-c-to-milliseconds

Reply:
I think I found out why it is no longer in the specification, as described in [this reference](https://man7.org/linux/man-pages/man3/usleep.3.html)
`     POSIX.1-2001 declares this function
       obsolete; use nanosleep(2) instead.  POSIX.1-2008 removes the
       specification of usleep().`
When it comes to incorrect sleep return type I would suggest creating an issue, so we don't forget about that. We should change it in the next major release (0.5.x)

Reply:
Done. https://github.com/scala-native/scala-native/issues/2395